homeassistant/components/izone/__init__.py

```diff
@@ -29,29 +28,28 @@
 
 async def async_setup(hass: HomeAssistant, config: ConfigType) -> bool:
     """Register the iZone component config."""
-    if not (conf := config.get(IZONE)):
-        return True
 
-    hass.data[DATA_CONFIG] = conf
+    # Check for manually added config, this may exclude some devices
+    if conf := config.get(IZONE):
+        hass.data[DATA_CONFIG] = conf
 
-    # Explicitly added in the config file, create a config entry.
-    hass.async_create_task(
-        hass.config_entries.flow.async_init(
-            IZONE, context={"source": config_entries.SOURCE_IMPORT}
-        )
-    )
```

Comment:
This import has disappeared from the new code.

I see it's a really old piece of code, but if it is on purpose, then I think it should be a separate PR and `CONFIG_SCHEMA` should be adjusted accordingly.

Reply:
That was done accidentally. Sorted.